include/uv-win.h

```diff
@@ -647,3 +647,17 @@ RB_HEAD(uv_timer_tree_s, uv_timer_s);
 #ifndef X_OK
 #define X_OK 1
 #endif
+
+#if _MSC_VER >= 1900
+#undef stdin
```

Comment:
LNK4098 means you're mixing debug/non-debug or single-threaded/multi-threaded CRT libraries.  Like @saghul already said, that's not an issue with libuv itself and this issue tracker really is not the right place to discuss it.  If you want to pursue this, please move it to the mailing list.